internal/controlplane/server.go

```diff
@@ -339,14 +339,18 @@ func (s *Server) StartHTTPServer(ctx context.Context) error {
 	mw := otelhttp.NewMiddleware("webhook")
 
 	// Explicitly handle HTTP only requests
-	err := gwmux.HandlePath(http.MethodGet, "/api/v1/auth/callback/{provider}/cli", s.HandleProviderCallback())
+	err := gwmux.HandlePath(http.MethodGet, "/api/v1/auth/callback/{provider}/cli", s.HandleOAuthCallback())
 	if err != nil {
 		return fmt.Errorf("failed to register provider callback handler: %w", err)
 	}
-	err = gwmux.HandlePath(http.MethodGet, "/api/v1/auth/callback/{provider}/web", s.HandleProviderCallback())
+	err = gwmux.HandlePath(http.MethodGet, "/api/v1/auth/callback/{provider}/web", s.HandleOAuthCallback())
 	if err != nil {
 		return fmt.Errorf("failed to register provider callback handler: %w", err)
 	}
+	err = gwmux.HandlePath(http.MethodGet, "/api/v1/auth/callback/{provider}/app", s.HandleGitHubAppCallback())
```

Comment:
This is just (part of) a string we put in to the GitHub API, right?

Reply:
What I mean is that we don't need this to link up with anything else except our docs.

Reply:
We also need to put it in our `server-config.yaml` because the oauth client library that we use requires a callback URL and it is required to match the callback for the GitHub App.

Reply:
Ah, okay.  I was trying to figure out if this should share constants, and now I know what to tie together.